docs/users/extend/custom-commands.md

```diff
@@ -8,8 +8,6 @@ There are example commands provided in `ddev/commands/*/*.example` that can just
 
 * The command filename is not what determines the name of the command.  That comes from the Usage doc line (`## Usage: commandname`).
```

Comment:
@rfay  Just saw this line and from reading the code it's not checked the user provides this value and we also do not provide a default value here. So if this sentence is correct a command without usage should / must fail.

Reply:
I had a short look at the cobra command code and this works like written here. So I suggest we use the file name as default value. What do you think? Other solution would be we introduce a validation like done for the flags which would be a good long term solution but is a little bit more work ofc. If you don't mind I will prepare a solution here and clean up the whole part in commands.go.

Reply:
The filename is already the default. If you drop a random shell script into .ddev/commands/web it will show up with that name.

Reply:
Ah okey, so the docs are not fully correct here. But I think we can leave it untouched for the moment.